common/gpu/amd/default.nix

```diff
@@ -6,11 +6,6 @@
   ) // {
     default = true;
   };
-  options.hardware.amdgpu.opencl = lib.mkEnableOption (lib.mdDoc
-    "rocm opencl runtime (Install rocmPackages.clr and rocmPackages.clr.icd)"
-  ) // {
-    default = true;
```

Comment:
Once we have the option, we maybe should default this to true?

Reply:
For this module? Yeah I think it should use 
```nix
hardware.amdgpu.rocm.enable = lib.mkDefault true;
```

Reply:
Wasn't this called `hardware.amdgpu.opencl.enable = lib.mkDefault true;` now?

Reply:
yeah, sorry, that was a mistake on my part.